src/lib.rs

```diff
@@ -131,7 +131,11 @@ impl Neo4j {
         &self.pass
     }
 
-    // Return the connection URI to connect to the Neo4j server over IPv4.
+    /// Return the connection URI to connect to the Neo4j server via Bolt over IPv4.
```

Comment:
Ah, good catch about the rustdoc, I think I need to update this in a few other places as well 😅